blockly/src/entities/BlocklyMonster.java

```diff
@@ -63,7 +63,15 @@ public enum BlocklyMonster {
           new StraightRangeAI(
               6,
               new Skill(
-                  new InevitableFireballSkill(EntityUtils::getHeroPosition),
+                  new InevitableFireballSkill(
+                      // Adjust postion of the target so the fireball flies in a straight line
+                      () ->
+                          Game.hero()
+                              .flatMap(hero -> hero.fetch(PositionComponent.class))
+                              .map(pos -> Game.tileAT(pos.position()))
+                              .map(Tile::position)
+                              .map(point -> point.add(1, 1))
+                              .orElse(null)),
```

Comment:
@AMatutat Das ist ein Punkt, an dem ich etwas Schmerzen habe. Konzeptionell fände ich es besser, wenn hier ein `Optional` produziert würde und der `InevitableFireballSkill ` schauen muss, wie es mit `Optional.empty` umgehen will. Ich hatte für diesen PR nix gesagt, weil der hier gezeigte Code dem Konzept in `EntityUtils::getHeroPosition` folgt ... 

Das wäre eine prima Aufgabe für die Praxisphasen.